examples/fishjam-chat/screens/ConnectWithVideoRoomScreen.tsx

```diff
@@ -0,0 +1,186 @@
+
+      setRoomName(storedRoomName);
+      setUserName(storedUserName);
+      setEnv(storedEnv);
+    }
+    readData();
+  }, []);
+
+  const onTapConnectButton = async () => {
+    try {
+      setConnectionError(null);
+      setLoading(true);
+      const roomManagerUrl =
+        env === 'staging'
+          ? process.env.EXPO_PUBLIC_VIDEOROOM_STAGING_ROOM_MANAGER!
+          : process.env.EXPO_PUBLIC_VIDEOROOM_PRODUCTION_ROOM_MANAGER!;
+      saveStorageData({ env, roomName, userName });
+
+      const { fishjamUrl, token } = await joinRoomWithRoomManager(
+        roomManagerUrl,
+        roomName,
+        userName,
+      );
+
+      navigation.navigate('Preview', {
+        userName,
+        fishjamUrl,
+        peerToken: token,
+      });
+    } catch (e) {
+      const message =
+        'message' in (e as Error) ? (e as Error).message : 'Unknown error';
+      setConnectionError(message);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  return (
+    <DismissKeyboard>
+      <SafeAreaView style={styles.safeArea}>
+        <KeyboardAvoidingView behavior="height" style={styles.container}>
+          {connectionError && (
+            <Text style={styles.errorMessage}>{connectionError}</Text>
+          )}
+          <Image
+            style={styles.logo}
+            source={require('../assets/fishjam-logo.png')}
+            resizeMode="contain"
+          />
+          <View
+            style={{
+              flexDirection: 'row',
+              justifyContent: 'space-around',
+              rowGap: 10,
+            }}>
+            <Button
+              title="Staging"
+              type={env === 'staging' ? 'primary' : 'secondary'}
+              onPress={() => setEnv('staging')}
+              disabled={loading}
+            />
+            <Button
+              title="Production"
+              type={env === 'prod' ? 'primary' : 'secondary'}
+              onPress={() => setEnv('prod')}
+              disabled={loading}
+            />
+          </View>
+          <TextInput
+            onChangeText={setRoomName}
+            placeholder="Room Name"
+            defaultValue={roomName}
+          />
+          <TextInput
+            onChangeText={setUserName}
+            placeholder="User Name"
+            defaultValue={userName}
+          />
+          <Button
+            title="Connect"
+            onPress={onTapConnectButton}
+            disabled={loading}
+          />
+        </KeyboardAvoidingView>
+      </SafeAreaView>
+    </DismissKeyboard>
+  );
+}
+
+const windowWidth = Dimensions.get('window').width;
+
+const styles = StyleSheet.create({
+  safeArea: {
+    flex: 1,
+    backgroundColor: '#BFE7F8',
+  },
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    backgroundColor: '#BFE7F8',
+    padding: 20,
+    gap: 24,
+  },
+  errorMessage: {
+    color: 'black',
+    textAlign: 'center',
+    margin: 25,
+    fontSize: 20,
+  },
+  logo: {
+    width: windowWidth - 40,
```

Comment:
**suggestion:** Why not just inline `Dimensions.get('window').width` here? It's not reused anywhere.

Reply:
It was just copy from other screen. But indeed - let's update it.